tests/test_ext_commands.py

```diff
@@ -1,6 +1,7 @@
 import collections
 import pathlib
 import sys
+import shutil
```

Comment:
Not sure how picky we are about adding new imports to the tests but this should be fine

Reply:
I'm not sure what you mean? Do you mean which imports go where? In general the order is stdlib imports first, then third party imports, then imports from your own stuff.